source/client-integration/create-client-configuration.rst

```diff
@@ -65,43 +67,46 @@ A client configuration includes all organization specific configuration and all
 
   ..  group-tab:: Java
 
-        The first step is to load the enterprise certificate (virksomhetssertifikat) through the :code:`KeyStoreConfig`. It can be created from a Java Key Store (JKS) or directly from a PKCS12-container, which is the usual format of an enterprise certificate. The latter is the recommended way of loading it if you have the certificate stored as a simple file:
+        The first step is to create a :code:`KeyStoreConfig` which loads the enterprise certificate that identifies you as a client. The recommended way is to initialize it from a PKCS12-container file, which is the usual format of an enterprise certificate:
 
         ..  code-block:: java
 
             KeyStoreConfig keyStoreConfig;
-            try (InputStream certificateStream = Files.newInputStream(Paths.get("/path/to/certificate.p12"))) {
+            try (InputStream p12Stream = Files.newInputStream(Paths.get("/path/to/certificate.p12"))) {
                 keyStoreConfig = KeyStoreConfig.fromOrganizationCertificate(
-                    certificateStream, "CertificatePassword"
+                        p12Stream, "CertificatePassword"
                 );
             }
 
-        If you have a Java Key Store file containing the organization certificate, it can be loaded in the following way:
+
+        Alternatively, if you use Java Key Store files for your certificates, it can be loaded in the following way:
 
         ..  code-block:: java
 
             KeyStoreConfig keyStoreConfig;
-            try (InputStream certificateStream = Files.newInputStream(Paths.get("/path/to/javakeystore.jks"))) {
+            try (InputStream jksStream = Files.newInputStream(Paths.get("/path/to/keystore.jks"))) {
                 keyStoreConfig = KeyStoreConfig.fromJavaKeyStore(
-                        certificateStream,
+                        jksStream,
                         "OrganizationCertificateAlias",
                         "KeyStorePassword",
                         "CertificatePassword"
                 );
             }
 
-        When the certificate has been loaded correctly, a :code:`ClientConfiguration` can be initialized. A trust store and service Uri needs to be set to properly connect. Please change the trust store and service Uri in the following example when connecting to our production environment.
+        When the certificate has been loaded correctly, the resulting :code:`KeyStoreConfig` is used to create a :code:`ClientConfiguration`.
+
+        If not explicitly configured, a :code:`ClientConfiguration` is specified to use the production API, so you likely want to specify which service environment you want to integrate with, to make sure you connect to ``ServiceEnvironment.STAGING`` when your application runs in any test environment, and ``ServiceEnvironment.PRODUCTION`` for your production environment.
```

Comment:
Har vel ikke så mye med dokumentasjonen å gjøre - og er vel for sent om det er i bruk rundt omkring, men hadde test vært en bedre default enn prod her? 🙈

Reply:
Ja, kan godt hende at test hadde vært en vel så bra "sensible  default", hvis det i det hele tatt er interessant med noen "default" overhodet.

Men det er hva det er.